### ISSUE TITLE
Rendering stops when: picking after adding non-pickable primitive and before render is called

### ISSUE THREAD

**Issue by bkuster** [AnalyticalGraphicsInc/cesium#6817]
##### What I'm trying to do
In an application I'm developing, `scene.pick` gets called _a lot_, eg. on mouse move.
Additionally, primitives are added to the `scene` at a similar frequency.
These primitves should _not_ be picked (`allowPicking: false`)

##### What happens
Some times, rendering stops, due to an error arising during rendering:
```
An error occurred while rendering.  Rendering has stopped.
undefined
TypeError: Cannot read property 'translucentCommand' of undefined
TypeError: Cannot read property 'translucentCommand' of undefined
    at executeTranslucentCommandsSortedMRT (https://cesiumjs.org/Cesium/Source/Scene/OIT.js:605:160)
    at OIT.executeCommands (https://cesiumjs.org/Cesium/Source/Scene/OIT.js:621:13)
    at scene._executeOITFunction (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:2230:32)
    at executeCommands (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:2461:13)
    at executeCommandsInViewport (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:2853:9)
    at updateAndExecuteCommands (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:2701:13)
    at render (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:3265:9)
    at tryAndCatchError (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:3285:13)
    at Scene.render (https://cesiumjs.org/Cesium/Source/Scene/Scene.js:3329:13)
```

##### What causes this to happen
As stated above, picking and adding primitives happens a lot. In some seldom cases,
primitives are added (async), their ready promise resolves, but _before_ they are rendered
the first time, they are drawn in a _pick_ pass. Since they do not have a `pickId`,
this line is executed in [Scene.js:1527](https://github.com/AnalyticalGraphicsInc/cesium/blob/40ba971f4f01493c27b0f44c737c7c89bdbc1242/Source/Scene/Scene.js#L1527)
```javascript
if (scene.frameState.passes.pick && !defined(command.pickId)) {
    return;
}
```
So now the draw command of the primitve is no longer dirty, _but_ it was never
fully created.

##### What could solve the issue
So I hacked in a working solution for my case:
```javascript
var oit = scene._oit;
if (scene.frameState.passes.pick && !defined(command.pickId)) {
    if (defined(oit) && oit.isSupported() && !defined(derivedCommands.oit)) {
        command.dirty = true;
    }
    return;
}
```
But I'm not sure if this is a) the right place to fix the issue or b) the right approach.
If this _is_ a working solution, I'm more then happy to open a PR.

##### How to reproduce
I've create a small Sandcastle: https://gist.github.com/bkuster/498b83a2bad638e5ed43f24110cdf89d

**Comment by lilleyse** [comment]
Maybe just removing the lines

```js
if (scene.frameState.passes.pick && !defined(command.pickId)) {
    return;
}
```

is the right move. It doesn't look like they are required but @bagnell can confirm.

**Comment by hpinkos** [comment]
Thanks for the detailed issue write up @bkuster!  I just ran into this problem yesterday but couldn't find a way to reproduce it.  This is super helpful =)

**Comment by lilleyse** [comment]
Slightly modified version of @bkuster's Sandcastle that I used for testing: [localhost:8080 link](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRRb5swEP4rVl5KpNRpp2ma0jQaa+lWaVGiJIv2EKly4CBWjY1sQ5ZN+e87zCBANx4M57vv7jv7OwqmScHhCJrcEwlH8gCG5ynduj3vKnTmg5KWcQn6ani3kztZIEpDaJlMBLidOJeh5UqSUAOzsNQ85ZYXYDwTgoQh+b2TBJ8GheWch2ZNKGVR5LU4NEm8v+DySUClYPXpWRpMhPTIpM37S8/dhrbhXdSqZlXD+7Au9ckbHI21Sh8h0YAdX9++u6E3I1Kt1x/c6z2uw9HbrAVoCz+flE6ZvSQOhOCZUTxa5zpmIfhZBkyXDdFtsNoEP16eFqu5v+nmO/cLMGs13+cWzIT8o6NQCaWbmg+l1T8/v87gOnQhXjueroJHeuT24IvswLwb+nE47JG6mB1+rGmpexdL0HVxV+HSuteBC6GOSx6+cplMSMyEgcp5HpYaPVcidRJDqVUSp85Eb6W8RKg90Agye9iAsX6CEjd2A1rjB4Kszi/BQiVMY58pDx9LxOc8jt3QuNLVVNRFUAVyneG1BQVI+5XJSJT7YJ9lllvfDYrXTIySc5UbmCsUeopLiphmXtrEaYbtNiEUZLRUhpc5XMej+gTXPQKbUwZ0vvi+Dl7mi21QjfB/5vRuMBpMjT0JmFUEPvE0U9qSXAuP0rGFNBMINON9Hr6CpaExZcLpuAZNI14QHt3vBr1/x25AQsGMQU+cC7Hmv2A3mE3HGN+BCcUivNQFToZgpzLkcDv7Vm1SSqdjNN+irFJiz3Qr4x8)

Just move the mouse around a lot before the scene loads, triggering a crash.

**Comment by hpinkos** [comment]
Thanks again for the bug report @bkuster!  This was fixed in #6837 and will be included in the Cesium 1.48 release available August 1st.

**Comment by kenstack-ltd** [comment]
I have same issue with mouse moving. It says "Worker error". I'm using Cesium version 1.115

<img width="1637" alt="Screenshot 2024-07-06 at 00 04 00" src="https://github.com/CesiumGS/cesium/assets/100079397/0e161c0d-13af-4741-8981-d600e7bdd
<img width="395" alt="Screenshot 2024-07-06 at 00 04 34" src="https://github.com/CesiumGS/cesium/assets/100079397/3cd670dc-8082-4d3f-8874-b32481a8575c">
d68">
